ambry-store/src/main/java/com/github/ambry/store/StorageManager.java

```diff
@@ -449,6 +450,22 @@ public boolean controlCompactionForBlobStore(PartitionId id, boolean enabled) {
     return diskManager != null && diskManager.controlCompactionForBlobStore(id, enabled);
   }
 
+  @Override
+  public ClusterParticipant getPrimaryClusterParticipant() {
```

Comment:
Can we pass cluster participant object to Bootstrap Controller.

Reply:
Good point.
While `AmbryServer` is passing cluster-participant while initiating `BootstrapController`, it passed `clusterParticipants` to `StorageManager` and a lot of other classes. 
Then,  `StorageManager` extracts primaryClusterParticipant exactly the same way as `AmbryServer` does. 

Looks like, it should be fine to just use the primaryClusterParticipant from `AmbryServer` directly.

Change incorporated.